### PR TITLE
Crowdfunding raise

### PR DESCRIPTION
**Pull request by yuliaMasliak**

### CODE REVIEW

File: Xtrades.XtradesMarketingPage/src/app/core/testimonials.service.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import { Injectable } from '@angular/core';
+import {HttpClient} from "@angular/common/http";
+import {LoadingService} from "@core/loading.service";
+import {map, Observable} from "rxjs";
+
+export interface TestimonialModel {
```
Comment (ArtyomYatskevich) [inline Xtrades.XtradesMarketingPage/src/app/core/testimonials.service.ts:6]: let's create a model folder and move it there

File: Xtrades.XtradesMarketingPage/src/app/core/testimonials.service.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import { Injectable } from '@angular/core';
+import {HttpClient} from "@angular/common/http";
+import {LoadingService} from "@core/loading.service";
```
Comment (ArtyomYatskevich) [inline Xtrades.XtradesMarketingPage/src/app/core/testimonials.service.ts:3]: it is not used, please remove it

File: Xtrades.XtradesMarketingPage/src/app/layouts/promo-layout/promo-layout.component.ts [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+@Component({
+  selector: 'app-promo-layout',
+  templateUrl: './promo-layout.component.html',
+  styleUrls: ['./promo-layout.component.scss'],
+})
+export class PromoLayoutComponent implements OnInit {
+  constructor() {}
+
+  ngOnInit(): void {}
```
Comment (ArtyomYatskevich) [inline Xtrades.XtradesMarketingPage/src/app/layouts/promo-layout/promo-layout.component.ts:11]: do you really need this empty method ?

File: Xtrades.XtradesMarketingPage/src/app/shared/components/text-slider/text-slider.component.ts [hunk 4]
```diff
@@ -15,6 +15,24 @@ SwiperCore.use([Navigation, Pagination, Scrollbar, Autoplay]);
...
+      maxRatio: 2,
+    },
+    speed: 1000,
+    navigation: {
+      nextEl: '.button-next',
+      prevEl: '.button-prev',
+    },
+  };
+  @Input() testimonials: any = [];
+  @Input() type = '';
```
Comment (ArtyomYatskevich) [inline Xtrades.XtradesMarketingPage/src/app/shared/components/text-slider/text-slider.component.ts:37]: It would be great to create 'enum' for this

**Review by ArtyomYatskevich — CHANGES_REQUESTED** [review]

File: Xtrades.XtradesMarketingPage/src/app/shared/components/text-slider/text-slider.component.html [hunk 5]
```diff
@@ -2,26 +2,70 @@
   <div class="content__swiper-container">
     <swiper class="swiper" [config]="config">
       <ng-template swiperSlide *ngFor="let item of testimonials">
-        <div class="slide column slider-container">
+        <div class="slide column slider-container" *ngIf="type !== 'promo'">
```
Comment (ArtyomYatskevich) [inline Xtrades.XtradesMarketingPage/src/app/shared/components/text-slider/text-slider.component.html:5]: please use enum here instead of this string

Comment (yuliaMasliak) [reply Xtrades.XtradesMarketingPage/src/app/shared/components/text-slider/text-slider.component.html:5]: Should I create a new variable for it in the controller?
like:
public currentSliderType = sliderType;  (our imported enum)

and then in the template use it as:
 *ngIf="type !== currentSliderType.promo"

Comment (ArtyomYatskevich) [reply Xtrades.XtradesMarketingPage/src/app/shared/components/text-slider/text-slider.component.html:5]: @yuliaMasliak  yep, in the component

**Review by ArtyomYatskevich — APPROVED** [review]
lgtm

**Pull request merged**